docs/csharp/language-reference/keywords/in-generic-modifier.md

```diff
@@ -13,6 +13,9 @@ For generic type parameters, the `in` keyword specifies that the type parameter
 
 Contravariance enables you to use a less derived type than that specified by the generic parameter. This allows for implicit conversion of classes that implement variant interfaces and implicit conversion of delegate types. Covariance and contravariance in generic type parameters are supported for reference types, but they are not supported for value types.
 
+> Variant interface
```

Comment:
Instead of adding this sentence, a better fix is to replace "variant interfaces" in the paragraph above with "contravariant interfaces"